src/collections/_documentation/accounts/pricing.md

```diff
@@ -60,12 +58,249 @@ If your projects have a high volume of events, you can control how many errors S
 
 `What happens when my trial expires?`
 
-You are downgraded to the developer plan, which has limited members and events.
+You are downgraded to your original plan, which has limited members and events. We'll be reminding you to choose whether to adopt the updated plans Sentry offers to take advantage of Performance Monitoring.
```

Comment:
suggestion: 

You will be downgraded to the same plan you were on prior to starting the trial. Before the trial ends, Sentry will provide you options to upgrade and continue to use our latest features like Performance Monitoring.

Reply:
Thanks.